system-tests/tests/smoke/cre/v2_vault_don_test_helpers.go

```diff
@@ -0,0 +1,71 @@
+package cre
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+
+	vault_helpers "github.com/smartcontractkit/chainlink-common/pkg/capabilities/actions/vault"
+	jsonrpc "github.com/smartcontractkit/chainlink-common/pkg/jsonrpc2"
+	"github.com/smartcontractkit/chainlink-testing-framework/framework"
+	"github.com/smartcontractkit/chainlink/v2/core/capabilities/vault/vaulttypes"
+)
+
+func FetchVaultPublicKey(t *testing.T, gatewayURL string) (publicKey string) {
```

Comment:
Can we keep this private unless there's some reason for it to be public? (I don't see anything in this PR but I may have missed it)

Reply:
Ah no then it's OK 👍